UnitsNet/UnitDefinitions/Flow.json

```diff
@@ -101,6 +173,18 @@
           "Abbreviations": [ "л/ч" ]
         }
       ]
+    },
+	{
+      "SingularName": "UsGallonsPerSecond",
+      "PluralName": "UsGallonsPerSecond",
+      "FromUnitToBaseFunc": "x/264.1720523581484",
+      "FromBaseToUnitFunc": "x*264.1720523581484",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "galUS/s" ]
```

Comment:
In volume we've written `gal (U.S.)`. Then `gal (U.S.)/s` would be consistent here, but that seems awkward. @angularsen what do you think?

Reply:
We actually display US Gallons as gal (US).
Takes less space and seems to be clear enough.
Also less awkward when used in Flow gal (US)/s

We also display US Survey Feet as ft (US) ... ft (US)/s

Reply:
It is awkward, but at least it is consistent. I think it's fine.